buddyboss/buddyboss-platform
# Repair tool offset should start with 0 fix

### All Submissions:






### Changes proposed in this Pull Request:

Fix the repair tool to always start with 0 offset.

### How to test the changes in this Pull Request:

1. Go to backend, BuddyBoss -> Tools
2. Run one task 
3. It should run perfectly.

### Other information:
